### PR TITLE
jboss/artificer-wildfly:1.0.0.Beta1

### PR DESCRIPTION
**Pull request by brmeyer**
jboss/artificer/wildfly82:1.0.0.Beta1

Notes:
- Artificer currently supports Wildfly 8 only.  9 will not work, due to Keycloak incompatibility.  So, for now, I'm having to use version-specific images, rather than "jboss/artificer/wildfly:*".
- The use of `-Dhibernate.hbm2ddl.auto=update` is temporary.  1.0.0.Beta2 (or Final) has a more elegant solution using the real DDL SQL.

### CODE REVIEW

**Comment by brmeyer** [comment]
@goldmann, let me know if this is missing anything!

One thing I'm not clear on: how do the deployments to https://registry.hub.docker.com/repos/jboss work?  Am I correct that you guys handle that?

File: wildfly82/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Artificer is 100% open source -- contributions are welcome!
+
+Visit [artificer.jboss.org](http://artificer.jboss.org) to learn more.
+
+## Usage
+
+To start Artificer
+
+    docker run -it -p 8080:8080 jboss/artificer/wildfly82[:tag]
```
Comment (goldmann) [inline wildfly82/README.md]: This is an invalid name for the image. Instead I would propose `jboss/atificer-wildfly:VERSION`.

Comment (brmeyer) [inline wildfly82/README.md]: Fair enough -- pushed an update to correct that.  Like I mentioned on my original note, the main reason I did that was that Artificer 1.0 does not currently support Wildfly 9.  I had thought that having Wildfly-version-specific images might be a good idea, but I suppose most users don't really care about the underlying Wildfly version...

**Comment by jorgemoralespou** [comment]
@brmeyer Brett, I would change dir structure from wildfly to artificer-wildfly for 2 reasons:
- It is easier to name image after directory name (**jboss/artificer-wildfly**)
- It is more consistent with rest of images (see **apiman** or **switchyard**)

But this is just my view ;-)

**Comment by brmeyer** [comment]
@jorgemoralespou, fair enough -- made the change.

@jorgemoralespou, @goldmann: Anything else needed to merge this and deploy to the repo?

File: artificer-wildfly/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+RUN cd $HOME \
+  && curl http://downloads.jboss.org/artificer/artificer-$ARTIFICER_VERSION.zip | bsdtar -xvf- \
+  && cd artificer-$ARTIFICER_VERSION \
+  && chmod 777 install.sh \
+  && ./install.sh -Ds-ramp-distro.choices.platform=1 -Ds-ramp-distro.choices.platform.jboss-wildfly-8.path=$HOME/wildfly -Dejb-jms.password=artificer1! \
+  && cp artificer-realm.json $HOME/wildfly \
+  && cd .. \
+  && rm -rf artificer*  
+
+CMD ["wildfly/bin/standalone.sh", "-b", "0.0.0.0", "-c", "standalone-full.xml", "-Dkeycloak.import=wildfly/artificer-realm.json", "-Dhibernate.hbm2ddl.auto=update"]
```
Comment (goldmann) [inline artificer-wildfly/Dockerfile]: I would provide here full path to `standalone.sh`.

Comment (brmeyer) [inline artificer-wildfly/Dockerfile]: Done

**Pull request merged**